x-pack/plugins/watcher/public/sections/watch_list/index.scss

```diff
@@ -0,0 +1,6 @@
+/**
+ * 1. Prevent inherited flexbox layout from compressing this element on IE.
+ */
+ .watchState__message {
```

Comment:
Minor nit: there's an extra space here before the class declaration. My editor did that to me too.

Reply:
fixed